src/nimblepkg/nimscriptsupport.nim

```diff
@@ -154,7 +154,7 @@ proc setupVM(graph: ModuleGraph; module: PSym; scriptName: string, flags: Flags)
   cbos copyFile:
     os.copyFile(getString(a, 0), getString(a, 1))
   cbos getLastModificationTime:
-    setResult(a, toSeconds(getLastModificationTime(getString(a, 0))))
+    setResult(a, seconds(getLastModificationTime(getString(a, 0))))
```

Comment:
This looks wrong,  `toUnix` is the replacement for `toSeconds`

Reply:
```toUnix```: https://github.com/nim-lang/Nim/blob/master/lib/pure/times.nim#L422